ral-facilities/operationsgateway
# DSEGOG-129 display user sessions

## Description
- A user should be able to save a user session and once it's save the session should appear in the drawer (Highlighted). The user should also be able to navigate between session. when a user session is selected it should become highlighted.  

## Testing instructions
- Check if it loads a session correctly
- Check that you can navigate between session without any side effects 
- Check that the list of session updates when a new session is added



## Agile board tracking
closes DSEGOG-129
